nautobot/dcim/views.py

```diff
@@ -1486,13 +1486,28 @@ class DeviceBayTemplateBulkDeleteView(generic.BulkDeleteView):
 class ModuleBayCommonViewSetMixin:
     """NautobotUIViewSet for ModuleBay views to handle templated create and bulk rename views."""
 
+    def update(self, request, *args, **kwargs):
+        message = format_html(
+            "<div>The value of the <strong>Position</strong> field will default to"
+            " the <strong>Name</strong> of the module bay unless supplied by the user.</div>"
+        )
+        messages.warning(request, message)
```

Comment:
If these are always present, it might be cleaner to just use add them directly to the edit view template file(s)?

Reply:
There is no specific ModuleBay edit view template currently. Seems like we are using the default edit view template.
Would it be worth the effort to create a new one just for this warning message?

Reply:
Might be - should be a real short/simple template inheriting from the default one for the most part.